mrchecker-framework-modules/mrchecker-selenium-module/src/main/java/com/capgemini/mrchecker/selenium/core/newDrivers/NewRemoteWebElement.java

```diff
@@ -1,63 +1,33 @@
 package com.capgemini.mrchecker.selenium.core.newDrivers;
 
+import static com.capgemini.mrchecker.selenium.core.newDrivers.DriverManager.getDriver;
+
 import java.util.ArrayList;
 import java.util.List;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import org.openqa.selenium.By;
-import org.openqa.selenium.ElementNotVisibleException;
-import org.openqa.selenium.NoSuchElementException;
-import org.openqa.selenium.StaleElementReferenceException;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.remote.FileDetector;
-import org.openqa.selenium.remote.RemoteWebDriver;
-import org.openqa.selenium.remote.RemoteWebElement;
-import org.openqa.selenium.remote.UselessFileDetector;
+import org.openqa.selenium.*;
```

Comment:
Please check if we need to import whole package. By default formatter should be set to 5 classes to import whole pkg, but some issues faced before.